dev_swift/05b_early_stopping.ipynb

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
<p>It would be great to have a short explanation of the `\Dense&lt;Float&gt;.allDifferentiableVariables` when you have a bit of time.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai_docs/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>